src/transaction.ts

```diff
@@ -4,15 +4,19 @@ export enum TransactionStatus {
   /**
    * The transaction was rejected by the blockchain node.
    */
-  Failed,
+  Failed = "failed",
```

Comment:
This is s a breaking change in TypeScript AFAIK.

Reply:
Indeed thanks, I'll revert. I changed for a idea that I did not implement.